x-pack/test/security_solution_api_integration/test_suites/siem_migrations/rules/trial_license_complete_tier/get.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from 'expect';
+import { v4 as uuidv4 } from 'uuid';
+import {
+  createMigrationRulesFromSavedObjects,
+  deleteAllMigrationRules,
+  getMigrationRuleSavedObject,
+  migrationRulesRouteHelpersFactory,
+} from '../../utils';
+import { FtrProviderContext } from '../../../../ftr_provider_context';
+
+export default ({ getService }: FtrProviderContext) => {
+  const es = getService('es');
+  const supertest = getService('supertest');
+  const migrationRulesRoutes = migrationRulesRouteHelpersFactory(supertest);
+
+  describe('@ess @serverless @serverlessQA Get API', () => {
+    beforeEach(async () => {
+      await deleteAllMigrationRules(es);
+    });
+
+    it('should fetch existing rules within specified migration', async () => {
+      // create a document
+      const migrationId = uuidv4();
+      const migrationRuleSO = getMigrationRuleSavedObject({ migration_id: migrationId });
+      await createMigrationRulesFromSavedObjects(es, [migrationRuleSO]);
```

Comment:
We don't use the saved objects framework for migration rules, they are regular documents in a regular index. These names are a bit confusing. Consider: 
```suggestion
      const migrationRuleDocument = getMigrationRuleDocument({ migration_id: migrationId });
      await createMigrationRules(es, [migrationRuleDocument]);
```

Reply:
done in https://github.com/elastic/kibana/pull/206822/commits/3b23a40c87495b83e2d94225524b3b15dc3f6b92